src/styles/body.scss

```diff
@@ -0,0 +1,8 @@
+body {
+  margin: 0;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used here. Make sure it is defined in your SCSS files, otherwise this will cause an error during compilation.